File: LAB2/KnapsackProblem/Item.cs
namespace KnapsackProblem
{
    public class Item
    {
        public int value;
        public int weight;
        public double ratio;

        public Item(int value, int weight)
        {
            this.value = value;
            this.weight = weight;
            ratio = value / (double)weight;
        }

        public override string ToString()
        {
            return "v:" + value + "     w:" + weight + "    r:" + ratio;
        }
    }
}

File: KnapsackJava/src/main/java/org/example/Item.java
package org.example;

public class Item {

    private final int id;
    private final int weight;
    private final int value;
    private final double ratio;

    public Item(int id, int weight, int value) {
        this.id = id;
        this.weight = weight;
        this.value = value;
        this.ratio = (double) value / (double) weight;;
    }

    public int getId() {
        return id;
    }

    public int getWeight() {
        return weight;
    }

    public int getValue() {
        return value;
    }

    public double getRatio() {
        return ratio;
    }
}
